### PR TITLE
[WFLY-12919] When exception is returned from server, WildFlyResponseClient hangs and client never receives the response.

### PR DESCRIPTION
**Pull request by dcihak**
Test for [WEJBHTTP-29](https://issues.jboss.org/browse/WEJBHTTP-29)

Test uses two deployments. One with the EJB client and one with the remote bean throwing exception.
Test calls the client deployment and the client invokes the remote bean. Test should not keep hanging.

### CODE REVIEW

**Comment by dcihak** [comment]
DO NOT MERGE. Waits for the next version of wildfly-http-client.

File: testsuite/integration/manualmode/src/test/java/org/jboss/as/test/manualmode/ejb/client/exception/ExceptionClientHangsTestCase.java
Comment (bstansberry) [inline testsuite/integration/manualmode/src/test/java/org/jboss/as/test/manualmode/ejb/client/exception/ExceptionClientHangsTestCase.java]: @dcihak Can you comment here why this test needs a server with assertions disabled? It's not obvious so that may help future maintainers to understand. This seems like a good spot to explain as someone looking at what the arq container is for will likely end up here.

Comment (dcihak) [reply testsuite/integration/manualmode/src/test/java/org/jboss/as/test/manualmode/ejb/client/exception/ExceptionClientHangsTestCase.java]: @bstansberry  Thanks for the review. Comment was added.

Comment (bstansberry) [reply testsuite/integration/manualmode/src/test/java/org/jboss/as/test/manualmode/ejb/client/exception/ExceptionClientHangsTestCase.java]: Thanks!

**Comment by bstansberry** [comment]
retest this please

**Review by darranl — DISMISSED** [review]
Can you please update the commit to reference a WFLY Jira issue.

**Comment by dcihak** [comment]
@darranl Commit updated with the reference to WFLY-12919.

**Comment by darranl** [comment]
@dcihak can you please double check the tests as the failures appear related.  Tracing the Jira issue it looks as though the required component upgrade should already be in.

**Comment by bstansberry** [comment]
retest this please

**Comment by bstansberry** [comment]
retest this please

**Comment by darranl** [comment]
I have just kicked off CI again although didn't see any changes since the last round of failures.  If this still fails we may need to close the PR and reopen once ready to address.

**Comment by darranl** [comment]
@fl4via FYI this PR is stale and needs to be closed, CI is still failing with errors directly related to the changes.  Did you want to have a look at fixing so a test case can be merged?

**Comment by darranl** [comment]
I am going to close this PR, we are now well over one year since it was originally submitted but have been unable to progress to the point it can be merged.  

At the moment this PR also has a merge conflict which would need to be addressed.

**Pull request closed without merging**
